src/reader.tsx

```diff
@@ -39,9 +38,9 @@ function Input() {
             <input
               autoFocus
               ref={inputRef}
-              type="password"
+              type="text"
```

Comment:
Using "text" input type stops password managers from saving the value. Keeping the functionality of hiding the text using css associated with hidden-text